lib/tasks/restore.rake

```diff
@@ -22,10 +22,13 @@ namespace :restore do
         attrs = object.attributes_before_type_cast.slice(*db_columns).except("search_column")
 
         columns = attrs.keys
-        values = attrs.values.map do |val|
+        values = attrs.map do |_key, val|
           case val
           when NilClass then "NULL"
-          when String, Time then "'#{val.inspect.delete_prefix('"').delete_suffix('"')}'"
+          when String, Time
```

Comment:
Hab das nicht testet, gehe aber davon aus, dass das getestet wurde und Api Methoden zu verwenden macht wahrscheinlich mehr Sinn als von Hand zu quoten.

Reply:
der ganze Rake-Task ist nur manuell getestet. In dem Fall waren einige values mit `'` im String, was für das String-Quoting das gezeigte Ergebnis hatte.

Da wir dan dem Task nur arbeiten, WENN mal ein Restore angefragt wird, wird das auch weiter so ne Baustelle mit einzelen Detailverbesserungen bleiben.